rag_experiment_accelerator/utils/logging.py

```diff
@@ -22,6 +22,9 @@ def get_logger(name: str) -> logging.Logger:
     global _cached_logging_level
     if _cached_logging_level is None:
         _cached_logging_level = os.getenv("LOGGING_LEVEL", "INFO").upper()
+    # Make sure the _cached_logging_level is not an empty string
```

Comment:
**[Severity: Major]** Consider validating the logging level against known values (e.g., 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL') to avoid runtime issues if the environment variable contains an invalid logging level.


---

**[peer-review-pal]** We are sorry for the intrusion. Feel free to ignore or delete the comments/issue with the tag - they are automatically generated by our [Hack project](https://hackbox.microsoft.com/hackathons/hackathon2024/project/81776) - repository **[Peer Review Pal](https://github.com/microsoft/peer-review-pal)**.
If you find them useful, want us to delete them, have any feedback at all or want to contribute - reach out to some of us: juliame@microsoft.com vadimkirilin@microsoft.com oteslenko@microsoft.com vimeena@microsoft.com
You can prevent future comments from appearing by adding **#prp-ignore** to your PR description.